packages/expo-router/src/__tests__/getId.test.ios.tsx

```diff
@@ -1,81 +1,65 @@
-import { RouteNode } from '../Route';
-import { generateDynamic } from '../getRoutesCore';
-import { createGetIdForRoute } from '../useScreens';
+import { getUniqueId } from '../useScreens';
```

Comment:
`createGetIdForRoute` used to return a `getId` function. Now we can just test the `getUniqueId` directly.

Input & Output should be the same